std/browser/_ops_shim.ts

```diff
@@ -0,0 +1,288 @@
+// Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
+
+import { errors } from "../../cli/js/errors.ts";
```

Comment:
These imports are WIP hacks?

Reply:
Hmm... if they provide working functionality, and we don't want to duplicate that, perhaps it's okay to link to them like:
```ts
import { errors } from "https://raw.githubusercontent.com/denoland/deno/0ffc99a61ddfa958a436beef0d003ecead630d0f/cli/js/errors.ts"
```

Reply:
No, there aren't...

My opinion is as long as `std` is co-located in the same repo as `cli` then they should be relative imports, as they will automatically be synced and tested with the version of the repo at that commit point.

Reply:
How are these resolved from `https://deno.land/std`? 😅

Reply:
🤦

Reply:
silly me

Reply:
ok, I think I fixed it

Reply:
Would it be better as a symlink? Or does deno.land/std not support that?

Reply:
@hayd Nope, it looks like https://raw.githubusercontent.com does not support symlinks (I had the same idea).